### DISCUSSION TITLE
Why is regolith-linux pointing to the old web-site?

### DISCUSSION THREAD

**Discussion by ivanskodje** [regolith-linux/regolith-desktop#1082] [Q&A] [answered]
Why is regolith-linux (https://github.com/regolith-linux) page linking to the old web-site?

![image](https://github.com/user-attachments/assets/9746be07-0c1b-444d-b6a8-7f8b1e8a702b)

**Comment by khos2ow** [comment, marked as answer]
Thank you for pointing out, it's corrected now.
